### ISSUE TITLE
Packet 241

### ISSUE THREAD

**Issue by sngo25** [issy123/agario-protocol#6]
Sent from server after client connected to the room, maybe it contains server version.

Example data:

`[241, 18, 118, 183, 49]`

**Comment by terjanq** [comment]
I don't think so. Server.version is included into first 255 packet (64 before).

Example:
new TextDecoder("utf-8").decode(new Uint8Array([255,43,0,0,0,240,28,64,111,99,12,67,0,95,196,192,166,251,58,43,213,10,196,192,57,128,183,117,37,124,173,64,206,153,123,195,172,203,173,64,0,0,0,0,50,46,49,46,54,0]))
"�+�@ocC_����:+�
��9��u%|�@Ι{ì˭@2.1.6"

I think that packet 241 is the first value to generate token in packet 16.

**Comment by wishstorage** [comment]
@terjanq , yes it is so
